uv.lock

```diff
@@ -1,4 +1,5 @@
 version = 1
+revision = 1
```

Comment:
Was this added just to resync between what's installed and what's specified in the pyproject dependencies? I don't think we need to keep it